src/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { Fragment } from "react";
+import { Link } from "react-router-dom";
+import "./breadcrumbs.css";
+
+export type BreadcrumbsProps = {
+  nodes: {
+    displayName: string;
+    path: string;
+    id: string;
+    isCurrent?: boolean;
+  }[];
+  /**
+   Can be an SVG or simple bit of text, even an emoji
+  */
+  separator?: string | React.SVGProps<SVGSVGElement> | null;
+};
+
+export function Breadcrumbs({ nodes, separator = ">" }: BreadcrumbsProps) {
```

Comment:
Could we use a less-technology specific term here - Like "crumbs" or something rather than "nodes"?
Perhaps we can come up with "more friendly" names for the keys inside the type too - but I think the existing ones are already generic and self-documenting enough for our re-use.

Reply:
Still a bit specific but `levels` is a common term for breadcrumb children since it references IA structures.